src/zjs_callbacks.c

```diff
@@ -214,6 +214,7 @@ int32_t zjs_add_callback_list(jerry_value_t js_func,
         new_cb->js->this = jerry_acquire_value(this);
         new_cb->js->pre = pre;
         new_cb->js->post = post;
+        new_cb->js->this = jerry_acquire_value(this);
```

Comment:
You reverted this with a later patch so make sure you fix it in this resubmitted verison. :)